app/controllers/blobs_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
     if File.write("received_files/" + filename, JSON.pretty_generate(json))
       json["document_id"] = uuid
       json["document_filename"] = filename
+      AdjustedBankTransaction.new(params["blob"].tempfile,uuid).call
```

Comment:
This probably should be wrapped in some type of `if` or other Error catching.

Reply:
I added exception handle with begin-rescue